src/OrbitBase/include/OrbitBase/Result.h

```diff
@@ -13,7 +15,10 @@
 class ErrorMessage final {
  public:
   ErrorMessage() = default;
-  explicit ErrorMessage(std::string message) : message_(std::move(message)) {}
+  explicit ErrorMessage(std::string message) : message_{std::move(message)} {}
+  template <typename... Args>
+  explicit constexpr ErrorMessage(const absl::FormatSpec<Args...>& format, const Args&... args)
+      : message_{absl::StrFormat(format, args...)} {}
```

Comment:
Is there a chance that this new constructor is considered over the `std::string`-constructor when converting from something else? Maybe we should require at least 2 arguments for this constructor (format string and one arg) to be on the safe side, WDYT?

Reply:
it does not seem to help, gcc is still unhappy.

Reply:
I don't see the problem directly either. I will have a look.